src/trackers/HUNO.py

```diff
@@ -0,0 +1,304 @@
+# -*- coding: utf-8 -*-
+# import discord
+import asyncio
+from torf import Torrent
+import requests
+from difflib import SequenceMatcher
+from termcolor import cprint
+import distutils.util
+import os
+from pprint import pprint
+
+# from pprint import pprint
+
+class HUNO():
+    """
+    Edit for Tracker:
+        Edit BASE.torrent with announce and source
+        Check for duplicates
+        Set type/category IDs
+        Upload
+    """
+    def __init__(self, config):
+        self.config = config
+        self.tracker = 'HUNO'
+        self.source_flag = 'HUNO'
+        self.search_url = 'https://hawke.uno/api/torrents/filter'
+        self.upload_url = 'https://hawke.uno/api/torrents/upload'
+        self.forum_link = 'https://hawke.uno/pages/1'
+        pass
+
+    async def upload(self, meta):
+        await self.edit_torrent(meta)
```

Comment:
please use the shared edit_torrent function
https://github.com/L4GSP1KE/Upload-Assistant/blob/f849e0b0e2e2ec444b42f15ec480e32122cecd76/src/trackers/AITHER.py#L33